src/trunk.c

```diff
@@ -8072,6 +8082,9 @@ trunk_print_branches_and_bundles(platform_log_handle *log_handle,
          // Generate marker line if curr branch is a sub-bundle's start branch
          if (branch_no == sb->start_branch) {
             uint16 filter_count = trunk_subbundle_filter_count(spl, node, sb);
+
+            platform_assert(sb->state != SB_STATE_UNINIT);
```

Comment:
- This change is debugging-purpose

Reply:
investigate trunk_subbundle_filter_count if this function changes the state. If not, it should be before the function call.